docs/databases/swift-kuery.html

```diff
@@ -154,91 +164,58 @@ <h2 class="heading-2"><span class="blue-text">Step 3:</span> Create a class to r
       <p>Our class needs to inherit from `Table` and have fields that match the column names of the table we created in the database.</p>
```

Comment:
I would like to see `Table` link to the apidoc for Table in SwiftKuery.
also... "fields that match the column names of the table we created in the database". presumably the types need to match too? 
I would also like to see "We must also provide the table name as a property." added to the text above.